inlong-manager/manager-dao/src/main/resources/mappers/OperationLogEntityMapper.xml

```diff
@@ -48,25 +51,52 @@
         from operation_log
         where id = #{id,jdbcType=INTEGER}
     </select>
+    <select id="selectByCondition" resultMap="BaseResultMap"
+            parameterType="org.apache.inlong.manager.pojo.operationLog.OperationLogRequest">
+        select
+        <include refid="Base_Column_List"/>
+        from operation_log
+        <where>
+            <if test="request.inlongGroupId != null and request.inlongGroupId != ''">
+                and inlong_group_id = #{request.inlongGroupId, jdbcType=VARCHAR}
+            </if>
+            <if test="request.inlongStreamId != null and request.inlongStreamId != ''">
+                and inlong_stream_id = #{request.inlongStreamId, jdbcType=VARCHAR}
+            </if>
+            <if test="request.operationType != null and request.operationType != ''">
+                and operation_type = #{request.operationType, jdbcType=VARCHAR}
+            </if>
+            <if test="request.operationTarget != null and request.operationTarget != ''">
+                and operation_target = #{request.operationTarget, jdbcType=VARCHAR}
+            </if>
+            <if test="request.startDate != null and request.startDate != '' and request.endDate != null and request.endDate != ''">
+                and request_time &gt;= #{request.startDate, jdbcType=VARCHAR}
```

Comment:
RequestTime type is timestamp.
Please confirm that there will be no performance issues here.

Reply:
There won't be any problems.